app/build.gradle

```diff
@@ -244,6 +244,7 @@ android {
             buildConfigField "String", "IMAGE_URL_BASE", "\"https://upload.beta.wmflabs.org/wikipedia/commons\""
             buildConfigField "String", "HOME_URL", "\"https://commons.wikimedia.beta.wmflabs.org/wiki/\""
             buildConfigField "String", "COMMONS_URL", "\"https://commons.wikimedia.beta.wmflabs.org\""
+            buildConfigField "String", "COMMONS_URL_SEARCH", "\"https://commons.wikimedia.org\""
```

Comment:
I don't think it is worth creating such a specificity just for beta. Just let beta use the beta server for search, and nevermind if beta search does not work perfectly.

Reply:
sure @nicolas-raoul will do that :)